src/test/java/LoginTests.java

```diff
@@ -6,31 +6,33 @@
 import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
 
-public class  LoginTests extends BaseTest1 {
-   private String invalidEmail="holost@gmail.com";
-   private By avatarLocator=By.xpath("//*[contains(@alt,'Avatar of')]");
+public class LoginTests extends BaseTest {
+    protected String myEmail = "holostenco.yuliya@gmail.com";
```

Comment:
Value of local variable is not used- `String myEmail = "holostenco.yuliya@gmail.com";`

Please, remove code redundancies